Jenkinsfile

```diff
@@ -95,6 +95,17 @@ parallel(
 					}
 
 					sh 'make bootstrap'
+
+					if (env.SDK_GO_REPO && env.SDK_SHA) {
+						echo "Updating local glide.yaml with controller-sdk-go repo '${env.SDK_GO_REPO}' and version '${env.SDK_SHA}'"
+						sh "perl -i -0pe 's/github\\.com\\/deis\\/controller-sdk-go\\n\\s+version:\\s+[a-f0-9]+/${env.SDK_GO_REPO}\\n  version: ${env.SDK_SHA}/' glide.yaml"
```

Comment:
Had prev been trying the groovy route of:
```
def pattern = ~/github.com\/deis\/controller-sdk-go\n\s+version:\s+[a-f0-9]+/
def g = new File('glide.yaml')
text = g.text
g.withWriter { w ->
  w << text.replaceAll(pattern, "${env.SDK_GO_REPO}\n  version: ${env.SDK_SHA}")
}
```
Which is acceptable groovy and runs great, but the Jenkins Secure Script Plugin warns not to approve use of all/part of it:
```
signature : new java.io.File java.lang.String Approving this signature may introduce a security vulnerability! You are advised to deny it.
```
(Can be seen [here](https://ci.deis.io/scriptApproval/) as well)